indra/newview/llappviewer.cpp

```diff
@@ -3980,6 +3981,7 @@ void LLAppViewer::fastQuit(S32 error_code)
     _exit(final_error_code);
```

Comment:
If you are doing _exit to not report to bugsplat, you also need to remove SecondLife.exec_marker and logout_marker. Otherwise viewer will report "previoues viewer sessuion crashed" to statiscs server and we will have a hard time tracing where those percents are comming from.